README.md

```diff
@@ -25,7 +25,7 @@ const schema = {
   required: [ 'PORT' ],
   properties: {
     PORT: {
-      type: 'string',
+      type: 'number',
```

Comment:
this error is repeated many times in the doc.
Could you fix those examples too?

Reply:
Sure, thanks for the note.